package.json

```diff
@@ -25,5 +25,8 @@
     "eslint": "^5.16.0",
     "stylelint": "^9.10.1",
     "stylelint-config-recommended": "^2.2.0"
+  },
+  "dependencies": {
+    "prettier": "^2.0.5"
```

Comment:
is this being used do we need to add a new dependency as we don't really run node only manually when checking lint.?

Reply:
ah nah that can be removed.